docs/index.md

```diff
@@ -10,7 +10,7 @@ page_keywords: docker, machine, amazonec2, azure, digitalocean, google, openstac
 > **Note**: Machine is currently in beta, so things are likely to change. We
 > don't recommend you use it in production yet.
 
-Machine makes it really easy to create Docker hosts on your computer, on cloud
+Machine makes it really simple to create Docker hosts on your computer, on cloud
```

Comment:
Spot on, Nathan. The best way to do this is just to remove the value judgement altogether. Value judgements don't really belong in tech docs. So... "Machine lets you create Docker hosts on your computer, ..."